### PR TITLE
Adding C_0 continuous periodicity to NURBS meshes [periodic-nurbs-v2-dev] 

### PR DESCRIPTION
**Pull request by IdoAkkerman**
This PR allows boundaries in a NURBS mesh to be connected with an other boundary.
Ideal for periodic BCs.

NOTE: other PR and branch[Periodic-nurbs-dev] is polluted with other PRs.

### CODE REVIEW

File: mesh/nurbs.cpp [hunk 1]
```diff
@@ -2629,7 +2877,8 @@ void NURBSExtension::LoadSolution(std::istream &input, GridFunction &sol) const
          {
             for (int i = 0; i < nx; i++)
             {
-               const int l = (kv.Size() == 2) ? p2g(i,j) : p2g(i,j,k);
+               const int ll = (kv.Size() == 2) ? p2g(i,j) : p2g(i,j,k);
+               const int l  = DofMap(l);
```
Comment (v-dobrev) [inline mesh/nurbs.cpp]: The above two lines generate warnings: `ll` is unused and `l` is used in its own initialization.

File: mesh/nurbs.cpp [hunk 2]
```diff
@@ -2664,7 +2913,8 @@ const
...
```
Comment (v-dobrev) [inline mesh/nurbs.cpp]: Similar to above: these two line generate warnings: `ll` is not used and `l` is used in its own initialization.

**Comment by tzanio** [comment]
@IdoAkkerman can you take a look and fix the above issues. We are ready to merge otherwise...

**Comment by IdoAkkerman** [comment]
Sorry for the late reply. I think the fix is self explainatory...

**Comment by tzanio** [comment]
Thanks Ido!

**Pull request merged**